### ISSUE TITLE
Fix rubocop failures causing CI to fail

### ISSUE THREAD

**Issue by eoinkelly** [ackama/rails-template#138]
Rubocop has updated and decided it doesn't like

```
config/puma.rb:7:25: C: Style/RedundantFetchBlock: Use fetch("RAILS_MAX_THREADS", 5) instead of fetch("RAILS_MAX_THREADS") { 5 }. (https://github.com/JuanitoFatas/fast-ruby#hashfetch-with-argument-vs-hashfetch--block-code)
max_threads_count = ENV.fetch("RAILS_MAX_THREADS") { 5 }
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
config/puma.rb:23:21: C: Style/RedundantFetchBlock: Use fetch("WEB_CONCURRENCY", 1) instead of fetch("WEB_CONCURRENCY") { 1 }. (https://github.com/JuanitoFatas/fast-ruby#hashfetch-with-argument-vs-hashfetch--block-code)
workers_count = ENV.fetch("WEB_CONCURRENCY") { 1 }.to_i
```
and can't auto fix it

**Comment by Rabid-Dan** [comment]
Theres a fix for this sitting in #135 which should hopefully get merged today

**Comment by Rabid-Dan** [comment]
Fixed by #139
